packages/pipeline/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@leafygreen-ui/pipeline",
+  "version": "1.0.0",
+  "description": "LeafyGreen UI Kit Pipeline",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+  },
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
```

Comment:
Please make sure to use the latest versions of the @leafygreen-ui components

Reply:
Yeah I think you guys cut a new release shortly after I had raised the PR. I will update to use the latest version of all dependencies.

Reply:
Resolved in 090a6f0